onnxruntime/test/onnx/main.cc

```diff
@@ -60,9 +62,13 @@ void usage() {
       OrtGetApiBase()->GetVersionString());
 }
 
-static TestTolerances LoadTestTolerances(bool enable_cuda, bool enable_openvino) {
+static TestTolerances LoadTestTolerances(bool enable_cuda, bool enable_openvino, bool useCustom, double atol, double rtol) {
```

Comment:
Then we should not have" bool enable_cuda, bool enable_openvino" here. The args can replace them. Otherwise, we might end up with having all EP names here, which would be super ugly.

Reply:
If you don't mind, later on I can create a PR to remove the unused code.

Reply:
let's do it separately, @snnn